crypto/digestible/signature/src/lib.rs

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2018-2020 MobileCoin Inc.
```

Comment:
Another way to organize the code in this module is, put this part in `mc-crypto-sig`, then I don't think you need to modify `digestible` crate or `keys` crate

Reply:
or is your idea to kill off the `mc-crypto-sig` crate?

Reply:
it seems like the signature stuff fits pretty naturally in its own box, none of those other things need to have the signature stuff bundled with them

Reply:
Then you end up violating the orphan rule with `signature::Signer` implementations on Ed25519, or you have code for doing signatures in both places.

Reply:
I see, thanks. I agree, the way it is now is the right way